articles/machine-learning/how-to-manage-quotas.md

```diff
@@ -95,12 +95,13 @@ The following table shows additional limits in the platform. Please reach out to
 | Parameter servers per node | 1 |
 
 <sup>1</sup> Maximum lifetime is the duration between when a run starts and when it finishes. Completed runs persist indefinitely. Data for runs not completed within the maximum lifetime is not accessible.
+
 <sup>2</sup> Jobs on a low-priority node can be preempted whenever there's a capacity constraint. We recommend that you implement checkpoints in your job.
 
 ### Azure Machine Learning managed online endpoints (preview)
 [!INCLUDE [preview disclaimer](../../includes/machine-learning-preview-generic-disclaimer.md)]
 
-Azure Machine Learning managed online endpoints have the following limits.
+Azure Machine Learning managed online endpoints have the following limits. Please reach out to the AzureML product team through a technical support ticket to request an exception. 
```

Comment:
We allow to request limits increase but not exception

Reply:
it's implies exception to the default limit. I copied this line from the compute section.